app/controllers/laa_references_controller.rb

```diff
@@ -88,4 +95,14 @@ def set_link_attempt
                       LinkAttempt.new
                     end
   end
+
+  def redirect_after_link
+    if link_laa_reference
+      redirect_to edit_defendant_path(id: defendant.id)
+      flash[:notice] = I18n.t('laa_reference.link.success')
+    else
+      redirect_to new_laa_reference_path(id: defendant.id)
```

Comment:
**not a blocker** as this did not go in this branch  but failure should render not redirect really, is there a problem in rendering
 [render and redirect](http://tutorials.jumpstartlab.com/topics/controllers/render_and_redirect.html)

Reply:
I agree and I don't think there is an issue with rendering, other than it will break some tests. Given time constraints can we ticket this for future sprint?